app/src/main/java/org/simple/clinic/teleconsultlog/teleconsultrecord/screen/TeleconsultRecordModel.kt

```diff
@@ -48,4 +49,8 @@ data class TeleconsultRecordModel(
   fun patientLoaded(patient: Patient): TeleconsultRecordModel {
     return copy(patient = patient)
   }
+
+  fun patientPrescriptionsCloned(): TeleconsultRecordModel {
```

Comment:
How about naming this `teleconsultRecordCreated()` instead?

Reply:
Reason: It's the `update` function that knows that cloning the prescriptions means that the teleconsult record has been created. The model here does not know anything about the prescriptions at all.

Reply:
Good point, renaming it 👍🏽